src/mixins/nr-affiliation-mixin.ts

```diff
@@ -244,19 +244,26 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
   }
 
   /**
-   * Create a draft numbered business based on selected business type (If applicable).
+   * Creates a draft numbered business based on specified legal type.
    * @param accountId Account ID of logged in user.
    * @param legalType The legal type of the business that's being created.
    */
   async createNumberedBusiness (accountId: number, legalType: CorpTypeCd): Promise<string> {
+    if (this.isContinuationIn) {
+      // convert regular legal type to continuation in legal type
+      switch (legalType) {
+        case CorpTypeCd.BC_COMPANY: legalType = CorpTypeCd.CONTINUE_IN; break
+        case CorpTypeCd.BENEFIT_COMPANY: legalType = CorpTypeCd.BEN_CONTINUE_IN; break
+        case CorpTypeCd.BC_CCC: legalType = CorpTypeCd.CCC_CONTINUE_IN; break
+        case CorpTypeCd.BC_ULC_COMPANY: legalType = CorpTypeCd.ULC_CONTINUE_IN; break
+        case CorpTypeCd.SOCIETY: legalType = CorpTypeCd.CONT_IN_SOCIETY; break
+      }
+    }
```

Comment:
So here's the gist of it. This is right where we create the draft filing in Legal API.

As far as Namerequest UI is concerned, up to this point, the entity type / corp type was CR/BC/CC/UL/SO.

Maybe you're wondering how this works for NRs? It's similar -- Namerequest UI uses CR/BC/CC/UL/CO, but then Namex API returns the new NR with a converted `entityType` (C/CBEN/CCC/CUL/CS).